tests/components/rollershutter/test_mqtt.py

```diff
@@ -0,0 +1,116 @@
+"""
```

Comment:
Can you add an `__init__.py` file to this directory so it gets picked up by py.test ?